src/material-experimental/mdc-menu/menu-item.html

```diff
@@ -0,0 +1,5 @@
+<ng-content></ng-content>
+<div class="mat-mdc-menu-ripple" matRipple
+     [matRippleDisabled]="disableRipple || disabled"
```

Comment:
Should we be doing that for both this and `material/menu` in that case? Otherwise the comparison wouldn't really be fair.

Reply:
I think it would be good if we switched to using the `RippleRenderer` throughout our library instead of the directive wrapped version. This pattern is something that has appeared more and more in our library (injectable service that contains all the logic and a directive the wraps it for user convenience).  and I think it makes sense to internally always depend on the injectable service, but expose the directive for users.

As far as fairness, I wouldn't say we've really gone through and systematically optimized most of our components for payload size (other than fixing things that are glaringly super bad). It would be good to do that, but I feel like the most important thing is that if we ask people to adopt some new thing, its not significantly bigger than the previous thing

Reply:
I spent some more time looking into it, but I'm not sure that the size benefits would be that great. Since the `MatMenuItem` has a binding for `disableRipple`, it means that we would have to duplicate all of the logic from the ripple directive that adds/removes the ripple event listeners based on whether ripples or the whole component is disabled. We'd also need an extra change hook so that reacts to the binding changing after init.